frontend/src/components/Search.tsx

```diff
@@ -0,0 +1,21 @@
+import clsx from "clsx";
+import { TextField } from ".";
+
+export interface SearchProps {
+        className?: string;
+}
+
+export function Search({
+        className,
+}: SearchProps) {
+    return (
+	    <div className="p-12">
+                {/* TODO: Add search icon */}
```

Comment:
Need to do this before merging. It might be easier to add a prop like `renderStartAdornment` to `TextField` than to try and get the icon to show up properly as a sibling.